domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Beregningsgrunnlag.kt

```diff
@@ -9,35 +9,66 @@ import no.nav.su.se.bakover.domain.beregning.fradrag.Fradrag
 import no.nav.su.se.bakover.domain.beregning.fradrag.FradragFactory
 import no.nav.su.se.bakover.domain.beregning.fradrag.FradragTilhører
 import no.nav.su.se.bakover.domain.beregning.fradrag.Fradragstype
+import no.nav.su.se.bakover.domain.grunnlag.Grunnlag
 
 internal data class Beregningsgrunnlag private constructor (
     val beregningsperiode: Periode,
     val fradrag: List<Fradrag>
 ) {
     companion object {
-        fun create(beregningsperiode: Periode, forventetInntektPerÅr: Double, fradragFraSaksbehandler: List<Fradrag>): Beregningsgrunnlag {
-            return tryCreate(beregningsperiode, forventetInntektPerÅr, fradragFraSaksbehandler).getOrHandle { throw IllegalArgumentException(it.toString()) }
+        fun create(
+            beregningsperiode: Periode,
+            uføregrunnlag: List<Grunnlag.Uføregrunnlag>,
+            fradragFraSaksbehandler: List<Fradrag>,
+        ): Beregningsgrunnlag {
+            return tryCreate(
+                beregningsperiode,
+                uføregrunnlag,
+                fradragFraSaksbehandler,
+            ).getOrHandle { throw IllegalArgumentException(it.toString()) }
         }
 
-        private fun tryCreate(beregningsperiode: Periode, forventetInntektPerÅr: Double, fradragFraSaksbehandler: List<Fradrag>): Either<UgyldigBeregningsgrunnlag, Beregningsgrunnlag> {
+        private fun tryCreate(
+            beregningsperiode: Periode,
+            uføregrunnlag: List<Grunnlag.Uføregrunnlag>,
+            fradragFraSaksbehandler: List<Fradrag>,
+        ): Either<UgyldigBeregningsgrunnlag, Beregningsgrunnlag> {
             val fradrag: List<Fradrag> = fradragFraSaksbehandler.plus(
-                FradragFactory.ny(
-                    type = Fradragstype.ForventetInntekt,
-                    månedsbeløp = forventetInntektPerÅr / 12.0,
-                    periode = beregningsperiode,
-                    utenlandskInntekt = null,
-                    tilhører = FradragTilhører.BRUKER
-                )
+                uføregrunnlag.map {
```

Comment:
Bør sannsynligvis validere at ingen av uføregrunnlagene overlapper i tid, samt at de er tilstøtende/dekker hele perioden (tror vi har noen funksjoner som støtter slikt i periode). Kan vurdere om dette er riktig plass for denne valideringen, eller om den hører hjemme lenger inn.